tests/test_wfirst.py

```diff
@@ -536,15 +536,43 @@ def test_wfirst_detectors():
     im_list = [im0_1,im0_2]*4
 
     im_1.applyPersistence(im_list,galsim.wfirst.persistence_coefficients)
-    galsim.wfirst.applyPersistence(im_2, im_list)
+    galsim.wfirst.applyPersistence(im_2, im_list, method='linear') #check the linear method
     assert im_2.scale == im_1.scale
     assert im_2.wcs == im_1.wcs
     assert im_2.dtype == im_1.dtype
     assert im_2.bounds == im_1.bounds
     np.testing.assert_array_equal(
         im_2.array, im_1.array,
         err_msg='Persistence results depend on function used.')
+
+    im_unit = galsim.Image(np.ones((2,2)), copy=True)
+    im_f = im_unit*0.0
+    im_f1 = im_unit*0.0
+    illuminatin_list = [1.E3, 1.E4, 4.E4, 4.99E4, 5.01E4, 1.E5, 1.0E6]
+    im_f_list = [x*im_unit for x in illuminatin_list]
+
+    galsim.wfirst.applyPersistence(im_f, im_f_list, method='fermi') #check fermi method
+    galsim.wfirst.applyPersistence(im_f1, im_f_list)
```

Comment:
Suggest moving this line down to just above where you do the check with it to keep the two things separate.

I.e. first check that method='fermi' works correctly by comparing to your explicit calculation.  (Thanks for that btw.  That looks good.)

And then check that omitting the method is equivalent.